components/candy-machine/UpdateCandyMachine.tsx

```diff
@@ -156,14 +162,14 @@ const UpdateCandyMachine: FC<{
                 setStatus('Candy Machine updated successfully!')
             }
         } catch (err) {
-            setErrorMessage('Candy Machine update was not successful, please re-run.')
+            populateNotificationError('updating', 'Candy Machine update was not successful, please re-run.')
         }
         setIsInteractingWithCM(false)
     }
 
     const fetchCandyMachine = async (): Promise<IFetchedCandyMachineConfig | undefined> => {
         if (candyMachineAccount && anchorWallet && connection) {
-            setErrorMessage('')
+            setError('')
```

Comment:
why not resetting the error right before the `if`? That way you might be able to reset each time. 
Also, what happens with you dont enter the `if`, I think we are missing an error here or something